drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/editor/menu/InsertMenuBuilderTest.java

```diff
@@ -66,213 +77,189 @@
+            add(new DTCellValue52(1));
+            add(new DTCellValue52("descr"));
+            add(new DTCellValue52("md"));
+        }});
+
+        uiModel.selectCell(0,
+                           2);
+
+        builder.onDecisionTableSelectedEvent(new DecisionTableSelectedEvent(dtPresenter));
+
+        assertTrue(builder.miAppendRow.getMenuItem().isEnabled());
+        assertTrue(builder.miInsertRowAbove.getMenuItem().isEnabled());
+        assertTrue(builder.miInsertRowBelow.getMenuItem().isEnabled());
+        assertTrue(builder.miInsertColumn.getMenuItem().isEnabled());
     }
 
     @Test
     public void testOnDecisionTableSelectedEventWithMultipleRowsSelected() {
-        model.getConditions().add( new Pattern52() {{
-            setFactType( "Fact" );
-            getChildColumns().add( new ConditionCol52() {{
-                setFactType( "Fact" );
-                setFactField( "field1" );
-                setFieldType( DataType.TYPE_STRING );
-                setOperator( "==" );
-            }} );
-        }} );
-        model.getData().add( new ArrayList<DTCellValue52>() {{
-            add( new DTCellValue52( 1 ) );
-            add( new DTCellValue52( "descr" ) );
-            add( new DTCellValue52( "md" ) );
-        }} );
-        model.getData().add( new ArrayList<DTCellValue52>() {{
-            add( new DTCellValue52( 1 ) );
-            add( new DTCellValue52( "descr" ) );
-            add( new DTCellValue52( "md" ) );
-        }} );
-
-        uiModel.selectCells( 0,
-                             2,
-                             1,
-                             2 );
-
-        builder.onDecisionTableSelectedEvent( new DecisionTableSelectedEvent( dtPresenter ) );
-
-        verify( view,
-                times( 1 ) ).enableAppendRowMenuItem( eq( true ) );
-        verify( view,
-                times( 1 ) ).enableInsertRowAboveMenuItem( eq( false ) );
-        verify( view,
-                times( 1 ) ).enableInsertRowBelowMenuItem( eq( false ) );
-        verify( view,
-                times( 1 ) ).enableAppendColumnMenuItem( eq( true ) );
+        model.getConditions().add(new Pattern52() {{
+            setFactType("Fact");
+            getChildColumns().add(new ConditionCol52() {{
+                setFactType("Fact");
+                setFactField("field1");
+                setFieldType(DataType.TYPE_STRING);
+                setOperator("==");
+            }});
+        }});
+        model.getData().add(new ArrayList<DTCellValue52>() {{
+            add(new DTCellValue52(1));
+            add(new DTCellValue52("descr"));
+            add(new DTCellValue52("md"));
+        }});
+        model.getData().add(new ArrayList<DTCellValue52>() {{
+            add(new DTCellValue52(1));
+            add(new DTCellValue52("descr"));
+            add(new DTCellValue52("md"));
+        }});
+
+        uiModel.selectCells(0,
+                            2,
+                            1,
+                            2);
+
```

Comment:
I really didn't want to chime but this is an example where our coding convention gets absurd. These are constants and there's no debugging or readability advantage. The latter might be subjective so nothing to be done here...